http/src/main/resources/reference.conf

```diff
@@ -719,14 +719,14 @@ clusterFiles {
 # so we need to ensure we don't use stale cache entries.
 runtimeDnsCache {
   cacheExpiryTime = 5 seconds
-  cacheMaxSize = 100
+  cacheMaxSize = 10000
```

Comment:
I don't think this is necessary... this is prod metrics for this cache size
<img width="641" alt="Screenshot 2023-07-26 at 3 53 28 PM" src="https://github.com/DataBiosphere/leonardo/assets/32771737/8d0b47a6-ad80-4902-b23f-49f1883bd94d">

Reply:
Thank you! I'm going to bump this one to 1000 only